cdap-test/src/main/java/co/cask/cdap/test/DataSetManager.java

```diff
@@ -30,9 +30,7 @@
  * </p>
  *
  * <pre><code> 
- *   ApplicationManager appManager = deployApplication(AppWithTable.class);
- *   DataSetManager{@literal <}Table> myTableManager = appManager.getDataset("my_table");
- *   myTableManager = appManager.getDataset("my_table");
+ *   myTableManager = getDataset("my_table");
```

Comment:
The variable was being declared on line 34 (in original code). The Would be good just to have the declaration on this line now, since the original line 34 is deleted.

`DataSetManager{@literal <}Table> myTableManager = getDataset("my_table");`